packages/zowe-explorer/package.json

```diff
@@ -1958,7 +1958,7 @@
     "createDemoNodes": "ts-node ./scripts/createDemoNodes.ts"
   },
   "engines": {
-    "vscode": "^1.63.0"
+    "vscode": "^1.73.0"
```

Comment:
was just glancing over and notice this, changing to 1.79.0 for readonly capability in this [PR](https://github.com/zowe/vscode-extension-for-zowe/pull/2505/files)